nautobot/ipam/api/serializers.py

```diff
@@ -230,6 +230,32 @@ class Meta:
             "parent": {"required": False},
         }
 
+        detail_view_config = {
```

Comment:
Can you add "namespace",  "type" and "assigned" to `list_display_fields` on line 217 there?

Reply:
`namespace` and `assigned` would not work, as they are not serializer fields. This would be possible after #4355